homework5/task1.js

```diff
@@ -0,0 +1,16 @@
+let averageGrade = 80;
+
+
+if (averageGrade < 60 && averageGrade >= 0) {
+    console.log("Незадовільно");
+} else if (averageGrade >= 60 && averageGrade <= 70) {
```

Comment:
averageGrade >= 60 зайве, якщо твоя програма вже дійшла до цього іфа, то ця умова завжди буде правдивою
```suggestion
} else if (averageGrade <= 70) {
```